src/redux/reducers/ui.ts

```diff
@@ -222,6 +222,12 @@ export const uiSlice = createSlice({
     closeFolderExplorer: (state: Draft<UiState>) => {
       state.folderExplorer = {isOpen: false};
     },
+    closeKubeConfigBrowseSetting: (state: Draft<UiState>) => {
+      state.kubeConfigBrowseSetting = {isOpen: true};
```

Comment:
Can't really understand what is the meaning of this action 🤔 The name is suggesting a close of the `kubeConfigBrowseSettings` but you are setting `isOpen` to true...

Reply:
If you want to suggest me better naming convention then please do. (like instead of `isOpen` to `isClose` or any other)

I believe it is set the state, to check whether to close the `Drawer` or not. You can find the code in `App.tsx` component.